content/python/concepts/dunder-methods/terms/str/str.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '__str__()'
+Description: 'Returns a reader-friendly string representation of a class object.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Attributes'
+  - 'Classes'
+  - 'Functions'
+  - 'Methods'
+  - 'Objects'
+  - 'OOP'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`__str__()`** dunder method returns a reader-friendly string representation of a class object. It can be called with [`str()`](https://www.codecademy.com/resources/docs/python/built-in-functions/str) as well as the built-in function [`print()`](https://www.codecademy.com/resources/docs/python/built-in-functions/print). Unlike [`__repr__()`](https://www.codecademy.com/resources/docs/python/dunder-methods/repr), it is not necessary that **`__str__()`** return a valid python expression. 
```

Comment:
```suggestion
The **`__str__()`** dunder method returns a reader-friendly string representation of a class object. It can be called with [`str()`](https://www.codecademy.com/resources/docs/python/built-in-functions/str) as well as the built-in function [`print()`](https://www.codecademy.com/resources/docs/python/built-in-functions/print). Unlike [`__repr__()`](https://www.codecademy.com/resources/docs/python/dunder-methods/repr), it is not necessary that `__str__()` return a valid python expression. 
```

Just the first instance of the topic, concept, or term need to be in **bold** 😄